### PR TITLE
Changelog for 3813

### PR DESCRIPTION
**Pull request by BlackYps**

### CODE REVIEW

File: docs/_posts/2024-11-29-3813.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+layout: post
+title: Game version 3813
+permalink: changelog/3813
+---
+
+
+# Game version 3813 (29th of November, 2024)
+
+The release was delayed by a week because we found a desync issue on the last minute. I want to thank everybody that contributed to the search and fixing of the issue. Pinpointing a desync is especially difficult but through the combined efforts of many people we managed to solve this exceptionally quickly. I am really glad to have such a good team!
+This also shows that if you play on fafdevelop and encounter issues, it is very important that you actually report the issues to us. The desync issue was online on fafdevelop for a few weeks and we heard nothing. Testing and reporting issues is really crucial to ensure a smooth release.
```
Comment (Garanas) [inline docs/_posts/2024-11-29-3813.md:11]: I'd rephrase this. It's true, but it's not the responsibility of the player. Instead - perhaps turn it into a call to action? Something along the lines of:

_The desync issue was on FAF Develop for a few weeks. Want to contribute? Play on FAF Develop and/or FAF Beta Balance occasionally and report back to the game team. Whether all is good or you encountered issues - all data points are worth their effort._

File: docs/_posts/2024-11-29-3813.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+  There are further features ready such as a map preview that can help you prepare for the match ahead. That is not shown however until we have better tooling in place to gauge its impact.
+
+  From a developers perspective the matchmaker lobby is now maintainable. You can now start the matchmaker lobby locally through your development environment. This allows you to run the matchmaker lobby as if you would find a match in the client. The matchmaker lobby is build from the ground up with maintainability in mind. It now supports a hot reload-like functionality for the interface. This allows you to change the interface on the go, without having to relaunch the game.
+
+  All taken together this is still very much a work in progress and we would love to hear the feedback of the community. We welcome you on Discord in the dev-talk channel.
+
+- (#6499) Allow Footprint locking for (modded) units
```
Comment (Garanas) [inline docs/_posts/2024-11-29-3813.md:134]: Perhaps:

- (#6499) Introduce feature to generalize mechanic to switch footprints

File: docs/_posts/2024-11-29-3813.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+  Interested in how assembly patches work? Reach out to the game team. You can find us easiest via the official Discord server.
+
+- (#6533) Add support to assign hotkeys to secondary mouse buttons
+
+Their technical names are 'X Button 1' and 'X Button 2'. These are the buttons that are commonly found on the left or right side of a mouse.
+
+With thanks to RTD for the assembly patch.
+
+- (#6534) Select/go to commander hotkeys will now search for commanders in transports if it doesn't find any on the ground
```
Comment (Garanas) [inline docs/_posts/2024-11-29-3813.md:146]: I noticed that some entries have a full stop at the end, and others do not.

File: docs/_posts/2024-11-29-3813.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+    - `DoTTime` for FAF units is adjusted to the old actual DoT duration, so balance is not changed.
+    - Unit databases will now show the actual damage over time duration.
+
+- (#6467) Fix a crash during blueprint loading if a unit blueprint has an enhancements table but not an intel table.
+
+- (#6471) Fix Seraphim walls not orientating towards the terrain
+
+- (#6494) Fix a crash when the `EnableDiskWatch` command line argument is enabled.
+
+- (#6499, https://github.com/FAForever/FA-Binary-Patches/pull/94, https://github.com/FAForever/FA-Binary-Patches/pull/107) Fix hard crash when exiting the game when a Salem is on land
```
Comment (Garanas) [inline docs/_posts/2024-11-29-3813.md:215]: These long links won't work nice in-game.

**Comment by Garanas** [comment]
Do not forget to also populate the in-game changelog. There is a character width limit, make sure to compare it with other changelog files. Hopefully in the near future with https://github.com/FAForever/fa/pull/6473 this is no longer necessary to do manually 😄

**Comment by BlackYps** [comment]
The deployment documentation actually doesn't mention this step. I need more detailed instructions on this

File: docs/_posts/2024-11-29-3813.md (hunk 3 above)
Comment (BlackYps) [reply docs/_posts/2024-11-29-3813.md:146]: True, but I won't fix that now. We should decide on one system in the future though

File: docs/_posts/2024-11-29-3813.md (hunk 4 above)
Comment (BlackYps) [reply docs/_posts/2024-11-29-3813.md:215]: But they are required to make it work on github. I can remove them for the ingame changelog

**Comment by Garanas** [comment]
> The deployment documentation actually doesn't mention this step. I need more detailed instructions on this

This file represents the in-game changelog:

- https://github.com/FAForever/fa/blob/develop/lua/ui/lobby/changelogData.lua#L0-L1

Don't forget to update the `last_version` value there too. Essentially you create a new table entry that is roughly a copy of game version 3812, but then the name is `Developers patch` and the description is the content you have in the markdown file. There's a maximum width per line, so you'll need to add enters manually. If all is good, the changelog should pop up when you start the game and show you the latest changelog.

You could also keep the changelog relatively short and just reference people to go to GitHub. There's an in-game button for that. See also version game version 3801 at line 350. It mentions that the in-game changelog no longer necessarily includes all the details.
